### PR TITLE
Starting workflow for some graphs

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
Note....this is a VERY ugly silly graph, but just need to get it started.

### CODE REVIEW

File: scripts/process/clean_up_raw.R [hunk 1]
```diff
@@ -0,0 +1,12 @@
+process.clean_up_raw <- function(viz = as.viz("clean_up_raw")){
+  library(dplyr)
+  library(data.table)
+  
+  viz.data <- readDepends(viz)[["fetchGA_cida"]]
+  
+  viz.data <- viz.data %>%
+    mutate(date = as.Date(paste(year,month,day,sep="-"), origin = "1970-01-01")) %>%
```
Comment (ldecicco-USGS) [inline scripts/process/clean_up_raw.R]: So @wdwatkins I think I can take out this whole process, right?

Comment (wdwatkins) [inline scripts/process/clean_up_raw.R]: yeah I will add that in fetch.

Comment (ldecicco-USGS) [inline scripts/process/clean_up_raw.R]: I'm going to leave this in until your PR comes in actually (otherwise, my stuff here will break)

Comment (wdwatkins) [inline scripts/process/clean_up_raw.R]: I can remove it in my PR if you want

Comment (ldecicco-USGS) [inline scripts/process/clean_up_raw.R]: perfect

**Pull request merged**